README.md

```diff
@@ -8,22 +8,23 @@ Apache Drill is a distributed MPP query layer that supports SQL and alternative
 
 ## Developers
 
-Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md).
+Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md) and 
+also use slack channel for discussions.
 
 ## More Information
 Please see the [Apache Drill Website](http://drill.apache.org/) or the [Apache Drill Documentation](http://drill.apache.org/docs/) for more information including:
 
  * Remote Execution Installation Instructions
  * Information about how to submit logical and distributed physical plans
  * More example queries and sample data
- * Find out ways to be involved or disuss Drill
+ * Find out ways to be involved or discuss Drill
 
 
 ## Join the community!
-Apache Drill is an Apache Foundation project and is seeking all types of contributions.
+Apache Drill is an Apache Foundation project and is seeking all types of users and contributions.
 Please say hello on the [Apache Drill mailing list](http://drill.apache.org/mailinglists/)
-or join our [Google Hangouts](http://drill.apache.org/community-resources/) for more information.
-(More information can be found at the [Apache Drill website](http://drill.apache.org/)).
+or join our [Slack Channel](https://apache-drill.slack.com/) if you need help with using or developing Apache Drill.
```

Comment:
We have a public invite link for Slack which needs to be included as well. I'll look that up

Reply:
Maybe we could create a bit.ly link?

Reply:
Link: https://bit.ly/2VM0XS8

Reply:
So we could update to say, the Slack channel is https://apache-drill.slack.com, and to join, use the bit.ly link.